ChromaticHQ/drupalorg-slack
# Updated app to enable Jenkins triggers

## Description


## Motivation / Context
I had manually triggering working via `/dorank` but I wanted to be able to schedule a weekly post triggered by a POST from Jenkins.

## Testing Instructions / How This Has Been Tested
1. Run the [Jenkins job](https://jenkins.chromatic.is/job/Slack%20-%20Trigger%20Drupalorg%20Stats/) and see the integration post to the #sandbox channel.
1. Trigger an ephemeral update that only you will see with `/dorank`

## Screenshots
![image](https://user-images.githubusercontent.com/20355/80775044-90a86b80-8b2c-11ea-90f5-d063f5e5ba3e.png)

## PR Author Checklist
- [x] My PR description includes a reference to a ticket or other "motivation" for these changes.
- [x] My PR description includes details of how I tested these changes.
- [x] My PR description includes testing instructions for reviewers with links to the testing environment, if applicable.
- [ ] ~My code includes documentation updates to existing documentation or new sections if applicable.~
